eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -126,21 +128,21 @@ steps:
 
   - script: |
       chmod -R a+r .
-      ../dotnet.sh publish -bl:MauiAndroid.binlog -f net6.0-android -c Release
+      ../dotnet publish -bl:MauiAndroid.binlog -f net6.0-android -c Release
       mv ./bin/Release/net6.0-android/com.companyname.MauiTesting-Signed.apk ./MauiAndroidDefault.apk
     displayName: Build MAUI Android
     workingDirectory: $(Build.SourcesDirectory)/MauiTesting
 
   - script: |
       chmod -R a+r .
-      ../dotnet.sh publish -bl:MauiiOS.binlog -f net6.0-ios -c Release
+      ../dotnet build -bl:MauiiOS.binlog -f net6.0-ios -c Release
       mv ./bin/Release/net6.0-ios/iossimulator-x64/MauiTesting.app ./MauiiOSDefault.app
     displayName: Build MAUI iOS
     workingDirectory: $(Build.SourcesDirectory)/MauiTesting
 
   - script: |
       chmod -R a+r .
-      ../dotnet.sh publish -bl:MauiMacCatalyst.binlog -f net6.0-maccatalyst -c Release
+      ../dotnet publish -bl:MauiMacCatalyst.binlog -f net6.0-maccatalyst -c Release
```

Comment:
Do we care about maccatalyst size? I'm not certain this is an important scenario to track. It may just be wasting resources and cluttering the perf dashboard.